tests/e2e/onboarding.test.ts

```diff
@@ -127,6 +127,7 @@ test('onboarding with a short code', async ({ page }) => {
 test('login as existing user', async ({ page }) => {
 	const password = faker.internet.password()
 	const user = await insertNewUser({ password })
```

Comment:
This utility handles cleanup for us 😁

But I have noticed sometimes we're not cleaning up the users properly. So if you can figure out why the cleanup logic doesn't always work then I'll gladly merge a PR! But this one isn't the proper fix. Thanks though!